nexus/src/app/mod.rs

```diff
@@ -486,12 +480,6 @@ impl Nexus {
                     for task in task_nexus.background_tasks.driver.tasks() {
                         task_nexus.background_tasks.driver.activate(task);
                     }
-                    if let Err(e) = task_nexus
```

Comment:
To future readers: This was needed in order to backfill a new `rack_subnet` row in the CRDB `rack` table for existing customer racks, since the bootstore had that subnet information from wicket, but CRDB did not. This was needed for BGP. However, reading from the bootstore is a always suspect, as it is eventually consistent, and can return different results. This worked for this case since we require all nodes to ack during RSS, and because the `rack_subnet` is immutable. It does not work in the add sled case, because a new node would not have this information yet. This wouldn't actually be a problem in production, because the `rack_subnet` would not get overwritten after RSS on existing racks. But it is a problem in testing, given the ordering of our fake startup. 

Since this is no longer needed, and since reading from the bootstore is wrong in general, it is safe to remove.